linux/inventory.yml

```
@@ -40,10 +40,12 @@ splunk_base_image:
       hosts:
         HSSPLNKHF[01:99]:
         PPSPLNKHF[01:99]:
+        MLUSPLKHF01P[01:99]P:
```

Realizing there are some redundant characters in the hostname:

```suggestion
        MLUSPLKHF[01:99]P:
```